### PR TITLE
docs: add HeroDevs mentions to README and LTS docs

### PR DESCRIPTION
**Pull request by AndreAngelantoni**
This PR includes static file changes to:

- the README.md
- the Long Term Support page (LTS.md) 

#### Related Issues

Addresses https://github.com/fastify/fastify/issues/5623.

#### Checklist

I have reviewed the Contributor Guidelines.

- [X] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,6 +266,19 @@ application, you should __always__ benchmark if performance matters to you.
 Please visit [Fastify help](https://github.com/fastify/help) to view prior
 support issues and to ask new support questions.
 
+All previous versions of Fastify besides the latest are end of life and will not receive any security or bug fixes.
```
Comment (jsumners) [inline README.md:269]: This sentence is incorrect. The LTS doc clearly shows that v4 is still supported until 2025-04-30.

File: README.md [hunk 2]
```diff
@@ -266,6 +266,19 @@ application, you should __always__ benchmark if performance matters to you.
...
+
+Our partner, HeroDevs, provides commercial security fixes for all unsupported versions at https://herodevs.com/fastify-nes.
+
+| Version | Supported? | Commercial Support         |
+| :------ | :--------- | :------------------------: |
+| 5       | Yes        |                            |
+| 4       | Yes        |                            |
+| 3       | No         | [Available here][hd-link]  |
+| 2       | No         | [Available here][hd-link]  |
+| 1       | No         | [Available here][hd-link]  |
```
Comment (jsumners) [inline README.md:279]: Do we really need to replicate the LTS doc table here?

**Review by jsumners — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Eomm) [reply README.md:279]: Using the user's hat: I prefer a table over a text wall

**Review by Eomm — COMMENTED** [review]
COuld you fix the markdown linter?

File: README.md (hunk 2 above)
Comment (jsumners) [reply README.md:279]: I'm just not convinced it needs to be in two places. I think we could link to the LTS doc in the main readme, and expand the table there.

Comment (mcollina) [reply README.md:279]: I agree.

Comment (joeeames) [reply README.md:279]: Hey all, we can definitely change the table to just some simple text. 

We have operated with other projects on the assumption that users prefer simple tables with clear information without needing to click links to get full information, so we suggest this little table on all the projects we work with.

If you would prefer just text that's no problem. Please confirm which you want.

File: docs/Reference/LTS.md [hunk 3]
```diff
@@ -48,6 +48,11 @@ A "month" is defined as 30 consecutive days.
 > dependency as `"fastify": "~3.15.x"`. This will leave your application
 > vulnerable, so please use with caution.
 
+### Security Support Beyond LTS
+
+Our partner HeroDevs provides commercial security support through the OpenJS Ecosystem Sustainability Program for versions of Fastify that are beyond LTS. For more information, see their [Never Ending Support][hd-link] service.
```
Comment (Uzlopak) [inline docs/Reference/LTS.md:53]:
```suggestion
Our partner HeroDevs provides commercial security support through the OpenJS Ecosystem Sustainability Program for versions of Fastify that EOL. For more information, see their
[Never Ending Support][hd-link] service.
```

File: README.md (hunk 2 above)
Comment (AndreAngelantoni) [reply README.md:279]: Removing....

File: README.md (hunk 1 above)
Comment (AndreAngelantoni) [reply README.md:269]: Changing...

**Comment by AndreAngelantoni** [comment]
@jsumners @Uzlopak I think I've addressed all the requests and suggestions:
- support matrix is gone
- version support clarified to be "Version 3 and lower"
- link to our page is now [/support/](https://herodevs.com/support/fastify-nes) rather than /fastify-nes
- over-long lines are now within bounds
- borked links now unborked.

May I get another review?

**Review by gurgunday — APPROVED** [review]

**Review by jsumners — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -266,6 +266,12 @@ application, you should __always__ benchmark if performance matters to you.
 Please visit [Fastify help](https://github.com/fastify/help) to view prior
 support issues and to ask new support questions.
 
+Version 3 of Fastify and lower are EOL and will not receive any security or bug fixes.
+
+Our partner, HeroDevs, provides commercial security fixes for all unsupported versions
```
Comment (Fdawgs) [inline README.md:271]: https://fastify.dev/docs/latest/Guides/Style-Guide/

"You"/"our" is fine for any informal documentation in `docs/Guides` but not for anything in `docs/Reference`.

**Review by Fdawgs — CHANGES_REQUESTED** [review]
See comment.

File: docs/Reference/LTS.md [hunk 5]
```diff
@@ -48,6 +48,13 @@ A "month" is defined as 30 consecutive days.
 > dependency as `"fastify": "~3.15.x"`. This will leave your application
 > vulnerable, so please use with caution.
 
+### Security Support Beyond LTS
+
+Our partner HeroDevs provides commercial security support through the OpenJS Ecosystem
```
Comment (Fdawgs) [inline docs/Reference/LTS.md:53]:
```suggestion
Fastify's partner HeroDevs provides commercial security support through the OpenJS Ecosystem
```

File: README.md (hunk 4 above)
Comment (Fdawgs) [inline README.md:271]:
```suggestion
Fastify's partner, HeroDevs, provides commercial security fixes for all unsupported versions
```

File: README.md [hunk 6]
```diff
@@ -266,6 +266,12 @@ application, you should __always__ benchmark if performance matters to you.
...
+at [https://herodevs.com/support/fastify-nes][hd-link]. Our supported version matrix
+is available in our [Long Term Support][lts-link] documentation.
```
Comment (Fdawgs) [inline README.md:273]:
```suggestion
Fastify's partner, HeroDevs, provides commercial security fixes for all unsupported versions
at [https://herodevs.com/support/fastify-nes][hd-link]. Fastify's supported version matrix
is available in the [Long Term Support][lts-link] documentation.
```

**Review by Fdawgs — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (AndreAngelantoni) [reply README.md]: Corrected where needed.

File: docs/Reference/LTS.md (hunk 5 above)
Comment (AndreAngelantoni) [reply docs/Reference/LTS.md]: Modified.

**Pull request merged**